ruby/red-arrow/ext/arrow/raw-records.cpp

```diff
@@ -642,11 +642,35 @@ namespace red_arrow {
             auto record = rb_ary_new_capa(n_columns_);
             rb_ary_push(records_, record);
           }
+          row_offset_ = 0;
           for (int i = 0; i < n_columns_; ++i) {
             const auto array = record_batch.column(i).get();
             column_index_ = i;
             check_status(array->Accept(this),
-                         "[raw-records]");
+                         "[record-batch-raw-records]");
```

Comment:
Could you use `[record-batch][raw-records]`?
We use multiple tags to show nested contexts.